diff -up ncurses-5.7/misc/gen-pkgconfig.in.pkgconfig ncurses-5.7/misc/gen-pkgconfig.in
--- ncurses-5.7/misc/gen-pkgconfig.in.pkgconfig	2015-02-19 15:48:07.803773099 +0100
+++ ncurses-5.7/misc/gen-pkgconfig.in	2015-02-19 16:59:24.643458532 +0100
@@ -71,12 +71,15 @@ do
 	desc="ncurses @NCURSES_MAJOR@.@NCURSES_MINOR@"
 	if test $name = $MAIN_LIBRARY ; then
 		reqs=
+		privreqs=$TINFO_NAME
 		desc="$desc library"
 	elif test $name = $SUB_LIBRARY ; then
 		reqs=
+		privreqs=
 		desc="$desc terminal interface library"
 	else
 		reqs=$MAIN_LIBRARY
+		privreqs=$TINFO_NAME
 		desc="$desc add-on library"
 	fi
 
@@ -111,6 +114,7 @@ Name: $name
 Description: $desc
 Version: \${version}
 Requires: $reqs
+Requires.private: $privreqs
 Libs: $libs
 Cflags: $cflags
 EOF
